frontend/packages/dev-console/integration-tests/features/addFlow/add-page.feature

```diff
@@ -0,0 +1,64 @@
+@add-flow
+Feature: Add page on Developer Console
+              As a user, I should be able select a way to create an Application, component or service from one of the options provided on Add page
+
+    
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-add-page"
+              And user has installed OpenShift Serverless Operator
+              And user has installed OpenShift Pipelines operator
+              And user is at Add page
+
+        
+        @regression
+        Scenario: Getting started resources
+             Then user will see Getting started resources
+              And user will Create Application using Samples
+              And user will see Build with guided documentation
+              And user will see Explore new developer features
+
+
+        @regression
+        Scenario: Options to create an Application, Component and Service
+             Then user will see "Developer Catalog" card
+              And user will see "Git Repository" card
+              And user will see "Container Images" card
+              And user will see "Pipelines" card
+              And user will see "Samples" card
+              And user will see "From Local Machine" card
+              And user will see "Serverless" card
+        
+        
+        @regression
+        Scenario: Developer Catalog option to create an Application, Component and Service
+             Then user will see "All Services" option
+              And user will see "Database" option
+              And user will see "Operator Backed" option
+              And user will see "Helm Chart" option
+              And user will see "Event Source" option
+
+        
+        @regression
+        Scenario: Git Repository option to create an Application, Component and Service
+             Then user will see "From Git" option
+              And user will see "From Devfile" option
+              And user will see "From Dockerfile" option
+
+        
+        @regression
+        Scenario: From Local Machine option to create an Application, Component and Service
+             Then user will see "Import YAML" option
+              And user will see "Upload JAR file" option
+
+        
+        @regression
+        Scenario: Details Toggle on
+             When user keeps Details toggle on
+             Then user will see details of options on the card
+
+
+        @regression
+        Scenario: Details Toggle off
+             When user keeps Details toggle off
+             Then user will not see details of options on the card
```

Comment:
I would expect a test which focus more on the user.

For example:

```suggestion
        @regression
        Scenario: Details Toggle on
             When user enable Details toggle
             Then user will see label Details on
             Then user will see description "Browse the catalog to discover, deploy and connect to services"

        @regression
        Scenario: Details Toggle off
             When user disable Details toggle
             Then user will see label Details off
             Then user will not see description "Browse the catalog to discover, deploy and connect to services"
```

Reply:
Updated